### PR TITLE
chore: Use ConstraintLayout in fragment_explore_course_list.xml #142

### PR DESCRIPTION
**Pull request by MarinaTaran**
Change the fragment_explore_course_list layout to Constraint layout  issue #142

### CODE REVIEW

File: app/src/main/res/layout/fragment_explore.xml [hunk 1]
```diff
@@ -20,31 +33,28 @@
...
     </com.google.android.material.appbar.AppBarLayout>
+
     <com.google.android.material.tabs.TabLayout
         android:id="@+id/tabLayout"
         android:layout_width="match_parent"
         android:layout_height="?attr/actionBarSize"
-        android:layout_gravity="bottom"
-        app:tabTextAppearance="@android:style/TextAppearance.Widget.TabWidget"
         android:background="@color/colorPrimary"
-        app:tabSelectedTextColor="@color/colorAccent"
```
Comment (ujjwalagrawal17) [inline app/src/main/res/layout/fragment_explore.xml]: @MarinaTaran we need to make sure the textColo,r is same. Can you add the below lines ? 

```
        app:tabTextAppearance="@android:style/TextAppearance.Widget.TabWidget"
        app:tabSelectedTextColor="@color/colorAccent"

```

Comment (MarinaTaran) [reply app/src/main/res/layout/fragment_explore.xml]: ok.I'll add it

**Comment by MarinaTaran** [comment]
@ujjwalagrawal17  could you check changes in  the fragment_explore_course_list layout to Constraint layout in my commit " changed fragment_explore_course_list from linerlayout to ConstraintLayout"  issue #142

File: app/src/main/res/layout/fragment_explore.xml (hunk 1 above)
Comment (MarinaTaran) [reply app/src/main/res/layout/fragment_explore.xml]: we have in fragment_explore this lines

Comment (MarinaTaran) [reply app/src/main/res/layout/fragment_explore.xml]: pleas,check "code", we have this lines

**Comment by ujjwalagrawal17** [comment]
@MarinaTaran thanks for the PR !

**Comment by MarinaTaran** [comment]
@ujjwalagrawal17 thank you!

**Pull request merged**